publish.sh

```diff
@@ -39,6 +40,9 @@ publish lalrpop-util
 publish lalrpop-snap
 publish lalrpop
 
+git tag $VERSION
```

Comment:
You should create an [annotated tag](https://git-scm.com/docs/git-tag#_description) with `-a`.

Reply:
The current tags aren't annotated and they don't provide any benefit I can tell unless they are signed or given a message. The entire point of automating it is so Niko doesn't have to do manual steps and adding a default message seems pointless if it contains no more information than the tag itself.

Reply:
For some reason I was under the impression that lightweight tags cannot be pushed, but that is not true, they can. One small benefit of annotated tags is that `git describe` does not ignore them by default. `git describe` can be somewhat useful in automation, but as the version number in `Cargo.toml` needs to be updated anyway, I think you are right, there is little benefit in annotated tags for this use case.